File: readability/readability.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>
#include <ctype.h>
int length;
int vocab;
int sentences;
int index;
float L;
float S;

int main()
{
    char *paragraph = get_string("Enter your text: ");

    int l = 0;
    int v = 1;
    int s = 0;

    for (int i = 0; paragraph[i] != '\0'; i++)
    {

        if (isalpha(paragraph[i]))
        {
            l++;
        }

        else if (paragraph[i] == ' ')
        {
            v++;
        }

        else if (paragraph[i] == '.' || paragraph[i] == '!' || paragraph[i] == '?')
        {
           s++;
        }
    }

    length = l;
    vocab = v;
    sentences = s;

    L = 100.00 * (length /(float) vocab);
    S = 100.00 * (sentences /(float) vocab);

    index = round(0.0588 * L - 0.296 * S - 15.8);

    int n = index;

    if (n < 1)
    {
        printf("Before Grade 1\n");
    }

    else if (n >= 16)
    {
        printf("Grade 16+\n");
    }

    else
    {
        printf("Grade %i\n" , n);
    }

}

File: sentimental-readability/readability.py
# TODO

h = input("Enter the text:")

l = 0
v = 1
s = 0

i = 0

for i in range(len(h)):
    if str.isalpha(h[i]):
        l = l + 1

    elif h[i] == " ":
        v = v + 1

    elif h[i] == "." or h[i] == "?" or h[i] == "!":
        s = s + 1

L = 100 * (l / v)
S = 100 * (s / v)

index = round(0.0588 * L - 0.296 * S - 15.8)

if index < 1:
    print("Before Grade 1")

elif index >= 16:
    print("Grade 16+")

else:
    print(f"Grade", {index})
